Formula/bork.rb

```diff
@@ -6,9 +6,11 @@ class Bork < Formula
   license "Apache-2.0"
   head "https://github.com/skylarmacdonald/bork.git", branch: "main"
 
-  bottle :unneeded
-
   def install
+    %w[types/shells.sh types/pipsi.sh types/cask.sh test/type-pipsi.bats test/type-cask.bats].each do |f|
+      inreplace f, "/usr/local/", HOMEBREW_PREFIX
+    end
```

Comment:
```suggestion
    files = %w[types/shells.sh types/pipsi.sh types/cask.sh test/type-pipsi.bats test/type-cask.bats]
    inreplace files, "/usr/local/", HOMEBREW_PREFIX
```
I think `inreplace` can take an array as its first argument. Though

Reply:
Cool, thanks Carlo! If this `inreplace` solves the "ARM not identical to Intel" problem then I'll apply these syntax changes afterwards.